src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -438,6 +439,7 @@ public void Query()
 
         private void QueryHistory()
         {
+            // Using InvariantCulture since this is internal
 #pragma warning disable CA1308 // Normalize strings to uppercase
             var query = QueryText.ToLower(CultureInfo.InvariantCulture).Trim();
```

Comment:
The query here is received from user and is used in downstream comparisons using `CurrentCulture`. A conversion here using `InvariantCulture` might lead to loss of information. It would be great if you could please verify it.

Reply:
yes I agree here. I think we should use `CurrentCulture` as the query is user facing text.

Reply:
Thanks @somil55 @alekhyareddy28. Changing this to `CurrentCulture`